app/assets/stylesheets/automated_tests.scss

```diff
@@ -1,20 +1,11 @@
-@import 'common/core';
+@import 'common/_markus';
```

Comment:
Imported partial `common/_markus` should be written as `common/markus`<br>Prefer double-quoted strings